wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/en.md

```diff
@@ -0,0 +1,78 @@
+# Handling Foul Play
+_Main page: [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour)._
+
+
+## Why am I restricted?
+A user may get restricted if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
+-   Multi-accounting.
+-   Hacking or cheating (see below).
+
+
+## What is considered multi-accounting?
+Each user may only use **one** account. Creating new accounts is strictly forbidden. First offences will be punished with a silence of varying length depending on the number of accounts made. Users that continue creating new accounts despite previous warnings will be restricted.
+
+
+## What is considered cheating?
+The [rules](/wiki/Rules) that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player is considered cheating.
+
+This is a one-strike rule. Cheaters will be restricted on their first offence for an indefinite amount of time. The team may decide to lift their restriction if they successfully appeal.
+
+
+## I have been restricted for cheating, but I didn't cheat!
+osu! uses automated metrics combined with manual confirmation. If you don't know why you are restricted, contact [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+
+## Prove that I/they were cheating! Tell us how you know!
+The team will not provide any proof. Giving away the countermeasures would make them useless in further detecting cheaters.
+
+
+## I'll make a video to prove my innocence!
+Sorry, but if you have been restricted, the team is already sure you are cheating. A video will not help, because even these can be also doctored.
+
+
+## I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
+Stay calm and **don't discuss their plays in public chat**. Don't give them undue attention or ask others to spectate them.
+You should report the user and not mention them anywhere. There are three ways to do so.
+
+### User Profile
+This is the preferred way for reporting foul play because it provides more information overall.
+
+On the user's profile, a `Report User` button will appear when you move your cursor to their avatar. Write any information that can prove that the user is doing misbehaviour.
+
+*Take note of the selected game mode before reporting*
+
+### Beatmap Page
+In beatmap pages, you can report a user in the top leaderboards by clicking the `Report` button on the far right of their score.
```

Comment:
Mention that this is also as effective as using the profile page for voting, the reports go to the same endpoint.